client/beefy/src/worker.rs

```diff
@@ -22,20 +22,22 @@ use codec::{Codec, Decode, Encode};
 use futures::{future, FutureExt, StreamExt};
 use log::{debug, error, info, log_enabled, trace, warn};
 use parking_lot::Mutex;
+use tokio::time::{sleep, Duration};
```

Comment:
Could you replace this with `futures_timer::Delay`? We are currently only depending directly on tokio for the executor (and some tests).